lib/rules/sentence-case.js

```diff
@@ -1,9 +1,100 @@
+const phraseBank = new Set([
```

Comment:
love it! not that performance is a huge deal here, but a `Set` is great and v fast for this sort of operation

Reply:
will be implementing radix sort in my next linter just to keep the vibes going